contents/feedback/alerts/with-link-on-right/index.tsx

```diff
@@ -3,14 +3,13 @@ import type { FC } from "react"
 
 const WithLinkOnRight: FC = () => {
   return (
-    <Alert status="info" variant="subtle">
-      {/* FIXME: Iconをwarningに設定 */}
+    <Alert status="info" variant="subtle" maxW="5xl">
```

Comment:
Shouldn't `status` be `warning`, considering `FIXME`?

```diff
- <Alert status="info" variant="subtle" maxW="5xl">
+ <Alert status="warning" variant="subtle" maxW="5xl">
```

Reply:
Actually, I forgot to remove the comment, it should be `info`.

Reply:
I see. Probably the `FIXME` comment was wrong, right?

Reply:
I think so.

Reply:
I got it. Conversation resolved :)